ext/hook/uhook.c

```diff
@@ -250,6 +258,37 @@ static bool dd_uhook_begin(zend_ulong invocation, zend_execute_data *execute_dat
     }
     dyn->hook_data->execute_data = NULL;
 
+    if (dyn->hook_data->suppress_call) {
+        if (ZEND_USER_CODE(execute_data->func->type)) {
+            static union {
+                zend_op zop;
+                zval zv;
+            } retop[] = {[0].zop =
```

Comment:
What's this union for? zv seems not used except indefining the zop here... Did you mean struct? (so that .op1.constant points to it)

Reply:
`retop` has two elements. The first element is the zend_op and the second element is zval. The zval is the NULL that's the op1 constant for the zend_op. This way the address of the zval is after the address of the zend_op. If I used two statics it could happen that the zval would have a lower memory address, and then the offset would be negative. But zend_op.op1.constant is unsigned, so it cannot have negative values.

Reply:
Still, did you mean _struct_ instead of _union_? Because currently zop and zv overlap.